triton-client/src/components/App.tsx

```diff
@@ -35,67 +30,50 @@ function App() {
         if (isLoggedIn) dispatch(fetchProjects())
     }, [dispatch, isLoggedIn])
 
-    let userName
-    if (user) {
-        userName = `${user.firstName} ${user.lastName}`
-    } else {
-        userName = "UNKNOWN"
-    }
-
     return (
         <BrowserRouter>
-            <>
-                <Layout style={{ minHeight: "100vh" }}>
-                    <Sider
-                        collapsedWidth="0"
-                        breakpoint="lg"
-                        width="24rem"
-                        className="app-sidebar"
-                        theme="light"
-                    >
-                        <MGCHeader />
-                        {areProjectsLoading && (
-                            <Spin
-                                size="large"
-                                className="App-spinner"
-                                tip="Loading..."
-                            ></Spin>
-                        )}
-                        {projects && !areProjectsLoading && (
-                            <ProjectCardList projects={projects} />
-                        )}
-                        {projects.length === 0 && !areProjectsLoading && (
-                            <Alert
-                                message="No projects associated with this user."
-                                type="info"
+            <Layout style={{ minHeight: "100vh" }}>
+                <SiderMenu isOpened={collapsed} />
+                <Layout>
+                    <Content>
+                        <Button
+                            type="text"
+                            icon={
+                                collapsed ? (
+                                    <MenuUnfoldOutlined />
+                                ) : (
+                                    <MenuFoldOutlined />
+                                )
+                            }
+                            onClick={() => setCollapsed(!collapsed)}
+                            style={{
+                                fontSize: "16px",
+                                width: 64,
+                                height: 64,
+                            }}
+                        />
```

Comment:
I feel that the button takes up a bit too much space vertically which leaves a lot of whitespace above the project detail. Ideally it would be part of the sidebar.
![image](https://github.com/user-attachments/assets/1cfb576e-840f-475e-b0f0-54104dbc7334)

Reply:
we can talk about this tomorrow.